cityofaustin/atd-product
# 10674 home mobile view

Fixes cityofaustin/atd-data-tech/issues/10674

This PR branches off of #50 with mobile and tablet design implementations.

**Steps to test:**

[Netlify link](https://deploy-preview-51--atd-product.netlify.app)

1. First test on your normal laptop/desktop viewport.
2. Resize down the screen until the first breakpoint (tablet) where there should be two columns of information for both the We're here to help and Featured sections. Make sure the navbar and footer look good and work properly.
3. Resize down to the smallest breakpoint (mobile) and now everything should be in one column. Make sure the navbar and footer look good and work properly.
4. Extra points if you actually test with your phone and/or tablet
